cmd/kind-test/main.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
```

Comment:
This is simply testing of the internal kind library. This binary is a REPL that allows you to test setting up a cluster, adding nodes, deleting nodes, etc.